tests/x11/wireshark.pm

```diff
@@ -79,6 +120,9 @@ sub run {
     assert_screen "wireshark-capturing";
 
     # set filter
+    if ($wireshark_gui_version eq "qt") {
+        send_key "ctrl-/";
+    }
```

Comment:
that's certainly up to style but you can just use `send_key 'ctrl-/' if $wireshark_gui_version eq 'qt';`

Reply:
Good point; I followed your advice.